### PR TITLE
Complete requester MVP

### LINKED ISSUE #11: Requester: complete MVP

### PR DESCRIPTION
**Pull request by jorgecuesta**
This close #11

### CODE REVIEW

File: apps/go/requester/cmd/generate_sample.go [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+func GetPrompts(count int, task *types.Task, instance *types.Instance) []*types.Prompt {
+	prompts := make([]*types.Prompt, count)
+	for i := 0; i < count; i++ {
+		prompts[i] = GetPrompt(task, instance)
+	}
+	return prompts
+}
+
+func main() {
```
Comment (RawthiL) [inline apps/go/requester/cmd/generate_sample.go:81]: Is this file a test or debug? do we need this here?

File: apps/go/requester/config.sample.json [hunk 2]
```diff
@@ -7,16 +7,21 @@
...
+    "req_per_sec": 10,
+    "session_tolerance": 1
   },
   "log_level": "debug",
   "temporal": {
     "host": "localhost",
     "port": 7233,
     "namespace": "pocket-ml-testbench",
-    "task_queue": "requester-local"
+    "task_queue": "requester"
```
Comment (RawthiL) [inline apps/go/requester/config.sample.json:21]: if "evaluator" has its own section, shouldnt "requester" have also its own?

File: apps/go/requester/cmd/generate_sample.go (hunk 1 above)
Comment (jorgecuesta) [reply apps/go/requester/cmd/generate_sample.go:81]: thats only to generate data before sampler is done or to test relayer

File: apps/go/requester/config.sample.json (hunk 2 above)
Comment (jorgecuesta) [reply apps/go/requester/config.sample.json:21]: i will evaluator key inside of temporal

**Review by RawthiL — APPROVED** [review]

**Pull request merged**